metricq/history_client.py

```diff
@@ -571,8 +581,7 @@ async def history_raw_timeline(
             An iterator over values of this metric.
 
         Raises:
-            InvalidHistoryResponse:
-                if an invalid response was received
+            ~exceptions.InvalidHistoryResponse:
```

Comment:
It seems redundant, but every raises had a short description for when the exception is raised. I'd add it for consistency